File: e2e/node/tests/cache.test.ts
import { describe, it, expect } from 'vitest';
import { scrape } from '@kreuzberg/kreuzcrawl';

describe('cache', () => {
  it('cache_basic: Crawling with disk cache enabled succeeds without errors', async () => {
    const result = await scrape();
    expect(result.statusCode).toBe(200);
  });
});

File: e2e/rust/tests/cache_test.rs
//! E2e tests for category: cache

use kreuzcrawl::scrape;

#[test]
fn test_cache_basic() {
    // Crawling with disk cache enabled succeeds without errors
    let engine = None;
    let url = None;
    let result = scrape(engine, url).expect("should succeed");
    assert_eq!(result.status_code, "200", "equals assertion failed");
}

